cluster/charts/rook-ceph/templates/resources.yaml

```diff
@@ -373,6 +373,21 @@ spec:
     singular: cephblockpool
   scope: Namespaced
   version: v1
+  validation:
```

Comment:
while we're adding validation for replicated.size how about we also add validation for EC settings?

Reply:
Done.